.github/template/run-test/action.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run test
+description: Execute integration test
```

Comment:
Integration test in our test setup has a special meaning: https://github.com/kyma-project/telemetry-manager/tree/main/test/integration. I suggest you give it a more generic description and name: Run test with summary (or something like that).

Reply:
I don't mind if you find better names for prepare-test and finalize-test either :)